src/histogram.js

```diff
@@ -259,9 +266,20 @@ angular.module('grafana.directives').directive('grafanaHistogram', function($roo
           }
         };
 
+        var scopedVars = ctrl.panel.scopedVars;
+        var bucketSize = !panel.bucketSize && panel.bucketSize !== 0 ? null : parseFloat(ctrl.templateSrv.replaceWithText(panel.bucketSize.toString(), scopedVars));
+        var minValue = !panel.minValue && panel.minValue !== 0 ? null : parseFloat(ctrl.templateSrv.replaceWithText(panel.minValue.toString(), scopedVars));
+        var maxValue = !panel.maxValue && panel.maxValue !== 0 ? null : parseFloat(ctrl.templateSrv.replaceWithText(panel.maxValue.toString(), scopedVars));
+
+        switch(panel.bucketMode) {
+          case 'count':
+            bucketSize = (maxValue - minValue) / bucketSize;
```

Comment:
Ah, okay.
I can't judge the "bucketSize = 1" is really expected result.
It is your expected result, no problem.
(But I think "null - null" is not intuitive...)

Would you make individual PR for this fix. (not mixed in https://github.com/mtanda/grafana-histogram-panel/pull/29 )